content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-events/index.md

```diff
@@ -0,0 +1,950 @@
+---
+title : "Chainguard Enforce Events"
+lead: ""
+description: "Chainguard Enforce Events"
+type: "article"
+date: 2020-10-06T08:48:45+00:00
+lastmod: 2020-10-06T08:48:45+00:00
+draft: false
+images: []
+weight: 001
+---
+
+### Introduction
+
+Chainguard Enforce generates and emits [Cloudevents](https://cloudevents.io/) based on actions that occur within a Chainguard account, such as registering a Kubernetes cluster or creating an IAM invitation. Enforce also emits events when workloads or policies are changed in a cluster.
+
+See the [chainguard-dev/enforce-events](https://github.com/chainguard-dev/enforce-events) GitHub repository for two sample applications that demonstrate how to create Slack notifications, as well as how to open a GitHub issue from received events.
+
+To subscribe to Enforce events for your account, use the `chainctl` command like this:
+
+```
+chainctl events subscriptions create –group $YOUR_GROUP https://<Your webhook URL>
+```
+
+Once you are subscribed to Enforce events, you will start receiving HTTP POST requests. Each request has a common set of Cloudevent header fields, denoted by the `Ce-` prefix. The event body is encoded using JSON and will have two top-level keys, `actor` and `body`.
+
+The `actor` field is the identity of the actor in your Enforce account that triggered the event, such as a team member or a Kubernetes cluster. The `body` field contains the specific data about the event, for example the response status for an invite creation request, or a cluster delete request.
+
+The following list of services and methods show example HTTP headers and bodies for public facing Enforce events. 
+
+### Service: Admission - Namespace
```

Comment:
I just realized this draft doesn't have any h2s. You could move all these headers up one level (h3 -> h2, h4 -> h3)

Reply:
Funny, I just assumed the title was an h2, but you're right, will promote these all up a level.